zndraw/server/tasks.py

```diff
@@ -109,6 +116,73 @@ class Scene(BaseModel):
     con = get_client(url)
     print(f"emitting scene_schema to {target}")
     con.emit("celery:task:results", asdict(msg))
+    con.sleep(10)
+    con.disconnect()
+
+
+@shared_task
+def geometries_schema(url: str, target: str):
+    msg = CeleryTaskData(
+        target=target,
+        event="draw:schema",
+        data=Geometry.updated_schema(),
+    )
+    con = get_client(url)
+    print(f"emitting scene_schema to {target}")
+    con.emit("celery:task:results", asdict(msg))
+    con.sleep(10)
+    con.disconnect()
+
+
+@shared_task
+def analysis_schema(url: str, token: str):
+    vis = ZnDraw(url=url, token=token)
+
+    config = GlobalConfig.load()
+
+    cls = get_analysis_class(config.get_analysis_methods())
+
+    schema = cls.model_json_schema_from_atoms(vis[0])
+    hide_discriminator_field(schema)
+
+    msg = CeleryTaskData(
+        target=f"webclients_{vis.token}", event="analysis:schema", data=schema
+    )
+    con = get_client(url)
+    print(f"emitting analysis_schema to {vis.token}")
+    con.emit("celery:task:results", asdict(msg))
+    con.sleep(10)
+    con.disconnect()
+    vis.socket.disconnect()
```

Comment:
If we separated the default modifiers into a separate stream in the cache, we could use this pattern for the default modifiers. We don't need to check `vis.available` because we're creating a new `ZnDraw` instance each time